src/verb/execution_builder.rs

```diff
@@ -409,5 +409,10 @@ mod execution_builder_test {
 }
 
 fn path_to_string<P: AsRef<Path>>(path: P) -> String {
-    path.as_ref().to_string_lossy().to_string()
+    let is_bash_on_windows = true;
```

Comment:
Did you forget something here ?

Reply:
Yes, deliberately so! I have hardcoded it locally to allow me to test `broot` in Windows bash, and when I've done some testing I'll implement the best method.

This is more of a CTA for any similar bugs that I can fix in one shot.